modules/standalone-georepl-site-removal.adoc

```diff
@@ -0,0 +1,74 @@
+:_content-type: PROCEDURE
+[id="standalone-georepl-site-removal"]
+= Removing a geo-replicated site from your standalone {productname} deployment
+
+By using the following procedure, {productname} administrators can remove sites in a geo-replicated setup. 
+
+.Prerequisites 
+
+* You have configured {productname} geo-replication with at least two sites, for example, `usstorage` and `eustorage`. 
+* Each site has its own Organization, Repository, and image tags. 
+
+.Procedure 
+
+. Enter the following command to execute a shell inside of the database pod:
++
+[source,terminal]
+----
+# podman exec -it quay-postgresql-database-pod -- /bin/bash
+----
+
+. Enter psql by running the following command:
++
+[source,terminal]
+----
+bash-4.4$ psql
+----
+
+. Enter the following command to reveal a list of sites in your geo-replicated deployment:
++
+[source,terminal]
+----
+quay=# select * from imagestoragelocation;
+----
++
+.Example output
++
+[source,terminal]
+----
+ id |       name
+----+-------------------
+  1 | usstorage
+  2 | eustorage
+----
+
+. Enter the following command to exit the postgres CLI to re-enter bash-4.4:
++
+[source,terminal]
+----
+\q
+----
+
+. Enter the following command to permanently remove the `eustorage2` site:
++
+[IMPORTANT]
+====
+The following action cannot be undone. Use with caution.
+====
++
+[source,terminal]
+----
+bash-4.4$ python -m util.removelocation
+----
++
+.Example output
++
+[source,terminal]
+----
```

Comment:
When Quay Customers execute the command to remove site, although they will see the following error message, the site can be removed successfully.
https://issues.redhat.com/browse/PROJQUAY-5650